### PR TITLE
Simplify HAProxy config

### PR DESCRIPTION
**Pull request by hollow**
https://remerge.atlassian.net/browse/CORE-840

### CODE REVIEW

**Comment by remerge-hal** [comment]
This comment ensures that the correct Slack channel is notified after the team/project label `CORE` has been added to this pull request.

See [this comment](https://remerge.atlassian.net/browse/ITSUP-1973?focusedCommentId=90701) for details.

File: tasks/main.yml [hunk 1]
```diff
@@ -28,71 +28,12 @@
...
-  ansible.builtin.copy:
-    src: "{{ item.src }}"
-    dest: "/etc/haproxy/conf.d/{{ item.name }}.cfg"
-    owner: haproxy
-    group: haproxy
-    mode: "0644"
-  notify:
-    - Test Config
-    - Reload haproxy
-  with_items: "{{ haproxy_additional_host_configs }}"
```
Comment (tundeaoni) [inline tasks/main.yml:46]: Adds support for static configurations i.e so we don't only render configurations from consul. The options provided by the base role(from Roberto) doesnt allow for proper configuration customization

File: tasks/main.yml [hunk 2]
```diff
@@ -28,71 +28,12 @@
...
-- name: Delete unwanted files
-  ansible.builtin.file:
-    path: "{{ item.path }}"
-    state: absent
-  with_items: "{{ unwanted_files['files'] }}"
-  notify:
-    - Test Config
-    - Reload haproxy
-
-- name: Create log directory if it does not exist
```
[the remark this answers is not in the document]
Comment (hollow) [reply tasks/main.yml:69]: I have switched logging to stdout instead of /dev/log

File: tasks/main.yml (hunk 1 above)
Comment (hollow) [reply tasks/main.yml:46]: We don't use this anywhere though because everything is based on consul

**Pull request merged**